### ISSUE TITLE
Feature Request: An Refresh Stats button

### ISSUE THREAD

**Issue by jeff11111111** [SBBTracker/SBBTracker#43]
Hi,

I'm finding that my stats often aren't updating after games (I've tried re-attaching to SBB, restarting SBB, restarting SBB tracker and nothing seems to work). If there could be some sort of refresh button that will attempt to find the most recent stats that would be fantastic.

SBB tracker is fantastic so thank you for making it.

**Comment by andrewrasch** [comment]
Yeah so there's an intermittent bug with games not being recorded that I don't exactly have a handle on. Hopefully I'll be able to fix it sooner rather than later
